### ISSUE TITLE
Sharing my custom tools for Mixxx and its library

### ISSUE THREAD

**Issue by FrankwaP** [mixxxdj/mixxx#12330]
Hello! I've started a repo to regroup tools I use with Mixxx and its library: https://github.com/FrankwaP/mixxx-utils

Here are two interesting (hopefully)  tools that I've written in Python:

`fix_track_paths` is a Python tool to fix the tracks paths, using the informations from 
your music player database. It is usefull when the file has been renamed or if the track is 
now in another format (for example you wanted a better quality format and went from a mp3 to a flac file).

`mixxx_to_rekordbox_xml` does what it says but YES IT EXPORTS THE HOT CUES :-) 

Hope they might help someone! :-)

**Comment by FrankwaP** [comment]
UPDATE: the `mixxx_to_rekordbox_xml` now exports the playlists, as well as the "beatgrids" ("TEMPO" field in [rebordbox xml format](https://cdn.rekordbox.com/files/20200410160904/xml_format_list.pdf)  if you have a constant tempo and hot cues.

I tested the workflow (Mixxx -> mixxx_to_rekordbox_xml -> import in Rekordbox -> export to USB key)  yesterday and everything works fine!

**Comment by ronso0** [comment]
FYI, cross-posting from https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/export.20to.20rekordbox.20USB.20format

> There already is this WIP rust library I worked on with @Niko / Swiftb0y: https://github.com/Holzhaus/rekordcrate/pull/100
We still need to finalize the serialization code though. If you want to help out, we'd be happy to take contributions
If this works, we can integrate it into Mixxx.

**Comment by FrankwaP** [comment]
@merspieler Apparently one needs [a few tricks to make it work](https://erhan.es/blog/running-pioneer-rekordbox-on-linux/)  (which I haven't tried yet.)

**Comment by timewasternl** [comment]
> [@merspieler](https://github.com/merspieler) Apparently one needs [a few tricks to make it work](https://erhan.es/blog/running-pioneer-rekordbox-on-linux/) (which I haven't tried yet.)

So far, this has never been working for USB export yet unfortunately. Although I could see this working for creating a local Rekordbox database to export with one of the reverse engineered projects.

**Comment by FrankwaP** [comment]
> So far, this has never been working for USB export yet (due to not detecting USB devices) unfortunately. 

You're right! I have edited my original comment, as well as the README.md of my project.

> Although I could see this working for creating a local Rekordbox database to export with one of the reverse engineered projects.

Yes I've seen that you commented on `rex` repo :-)   
Did you find a way to export the hot cues and beatgrid with it? Because that's why I started my project in the first place.
